bundles/ui/org.eclipse.smarthome.ui/src/main/java/org/eclipse/smarthome/ui/internal/chart/ChartServlet.java

```diff
@@ -135,6 +132,16 @@ protected void deactivate() {
         httpService.unregister(SERVLET_NAME);
     }
 
+    protected void modified(Map<String, Object> config) {
```

Comment:
The attribute modified does not contain a default attribute:
* https://osgi.org/xmlns/scr/v1.2.0/scr.xsd
* <code>&lt;attribute name="modified" type="token" use="optional"/&gt;</code>

If we want (and I hope we want) to use an implementation that conforms to the specification (and not some implementation), we should fix that.

Do you read that scheme the same way or do I miss some point?